server/src/database/ quieres/index.ts

```diff
@@ -0,0 +1,4 @@
+import checkEmailQuery from './checkEmailQuery';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I understand you, but this command is temporary because we inside index.ts file and eslint needs to export the function as a default but in the future we will export many of functions